spec/controllers/index_spec.rb

```diff
@@ -0,0 +1,92 @@
+require_relative '../spec_helper.rb'
+
+describe 'index.rb' do
+#last_request
+let(:tracy) { Sealion.create(first_name: "Tracy", last_name: "Teague", age: 25, location: "New York, NY", quirk: "Puts ketchup on everything") }
+#-----GET------
+
+  describe '#get' do
+
+    it 'should get the /sealions route' do
+      get '/sealions'
+      expect(last_response).to be_ok
+
+    end
+
+    it 'should get a specific sealion from the database' do
+      get "/sealions/#{tracy.id}" #don't hardcode #always double quotes string interp
+      expect(last_response).to be_ok
+      expect(last_response.body).to include("Tracy Teague")
+    end
+
+  end
+
+#-----POST-----
+
+#   describe '#post' do
+
+# # Would I be able to use 'let' to create a new Sealion
+# # simply for testing to be available for the whole test?
+# # I hard-coded an id: 5 here for testing purposes
+
+#     it 'should post new information to the /sealions route' do
+#       post '/sealions', :params => {id: 5, first_name: "Link", last_name: "Blaine", age: 27, location: "New York, NY", quirk: "Making raindrop cheek noises."}
```

Comment:
Definitely don't hardcode expectations about `id`s. The `id`s that get assigned are technically up to the database. You can't specify the id of a created `sealion` by sending an `id` in `params`.